### DISCUSSION TITLE
开发者文档需求

### DISCUSSION THREAD

**Discussion by Taskeren** [TartaricAcid/TouhouLittleMaid#345] [Suggestions & Ideas]
求一个类似于 mezz 的 JustEnoghtItems 项目类似的开发者文档，列出 api 的 gradle 地址，方便开发。❤️

**Comment by ChloePrime** [comment]
javadoc秋梨膏

**Comment by TartaricAcid** [comment]
写了一个简单的参考案例：https://github.com/TartaricAcid/TLMAdditionExample
